style/review.less

```diff
@@ -0,0 +1,92 @@
+.review {
```

Comment:
Can you post a SS showing this styling?

Reply:
Here you go.

https://deploy-preview-65--coala-gh-board.netlify.com/#/r/coala:coala%7Ccoala-bears%7Cdocumentation%7Cprojects%7Cdevops%7Ccorobo%7Ccommunity%7Cgci-leaders%7Cgh-board%7Ccoala-html%7Clanding-frontend%7CcoalaCSS/kanban

Reply:
We need to fix https://github.com/coala/gh-board/issues/3 so people don't get confused.

Reply:
![jietu20180627-224450](https://user-images.githubusercontent.com/25746010/41981546-c098ee0c-7a5b-11e8-99fa-000cd80c366c.jpg)